### PR TITLE
Feature/1.2.1 to support

### PR DESCRIPTION
**Pull request by viniciusguardieirozup**
### Description and Example

Change the androidx to android support

### Checklist

Please, check if these important points are met using `[x]`:

- [x] I read the [PR Guide] and followed the process outlined there for submitting this PR.
- [x] I avoided _breaking changes_ by not changing public APIs that people rely on. 
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/94d6e56ed73e22da2b4376683407ca4f3cb46728/android/beagle/src/main/java/br/com/zup/beagle/android/data/serializer/PolymorphicJsonAdapterFactory.java#L309">android/beagle/src/main/java/br/com/zup/beagle/android/data/serializer/PolymorphicJsonAdapterFactory.java#L309</a> - Implicitly using the default locale is a common source of bugs: Use <code>toLowerCase(Locale)</code> instead. For strings meant to be internal use <code>Locale.ROOT</code>, otherwise <code>Locale.getDefault()</code>.</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle/blob/94d6e56ed73e22da2b4376683407ca4f3cb46728/android/beagle/src/main/java/br/com/zup/beagle/android/utils/ToolbarManager.kt#L216">android/beagle/src/main/java/br/com/zup/beagle/android/utils/ToolbarManager.kt#L216</a> - Prefer “<code>SHOW_AS_ACTION_IF_ROOM</code>” instead of “<code>SHOW_AS_ACTION_ALWAYS</code>”</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Review by hernandazevedozup — APPROVED** [review]

File: android/sample/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -91,7 +91,7 @@
         <item name="android:background">?attr/selectableItemBackground</item>
     </style>
 
-    <style name="DesignSystem.Text.Default" parent="Widget.AppCompat.TextView">
+    <style name="DesignSystem.Text.Default" parent="AppTheme">
```
Comment (matheusribeirozup) [inline android/sample/src/main/res/values/styles.xml:94]: Is the style of the text being inherited from the application style right?

Comment (viniciusguardieirozup) [reply android/sample/src/main/res/values/styles.xml:94]: This style is on sample, in the support library we don't have the Widget.AppCompat.TextView, so I decided to inherit the application style to this style, do you have some suggest to this inheritance?

Comment (matheusribeirozup) [reply android/sample/src/main/res/values/styles.xml:94]: The logic that applies the styles to the components of the beagle needs a style inherited from **its respective component**. I think that inheriting from the application will even cause a crash. I suggest you do the test the way you did, but change it to some textual component.

**Review by danilorochazup — APPROVED** [review]

**Pull request merged**